get_directory_sizes.py

```diff
@@ -13,11 +13,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Usage:
-    %(script)s [--verbose] [--num-workers=<n>] <artifactory-url> <repositories>...
+    %(script)s [--verbose] [--num-workers=<n>] [-u <username>] [-p <password>] <artifactory-url> <repositories>...
```

Comment:
I'd like these to be --username=<username> --password=<password> and have the -u and -p be in the options below, just like the --num-workers option.